doc/modules/preprocessing.rst

```diff
@@ -240,10 +240,65 @@ Centering kernel matrices
 -------------------------
 
 If you have a kernel matrix of a kernel :math:`K` that computes a dot product
-in a feature space defined by function :math:`\phi`,
-a :class:`KernelCenterer` can transform the kernel matrix
-so that it contains inner products in the feature space
-defined by :math:`\phi` followed by removal of the mean in that space.
+in a feature space (possibly implicitly) defined by a function
+:math:`\phi(\cdot)`, a :class:`KernelCenterer` can transform the kernel matrix
+so that it contains inner products in the feature space defined by :math:`\phi`
+followed by the removal of the mean in that space. In other words,
+:class:`KernelCenterer` computes the centered Gram matrix associated to a
+positive semidefinite kernel :math:`K`.
+
+**Mathematical formulation**
+
+We can have a look at the mathematical formulation now that we have the
+intuition. Let :math:`K` be a kernel matrix of shape `(n_samples, n_samples)`
+computed from :math:`X`, a data matrix of shape `(n_samples, n_features)`.
```

Comment:
Since `transform` is explicitly stated afterwards, we could maybe add a mention to `fit` here ? Something like

```suggestion
computed from :math:`X`, a data matrix of shape `(n_samples, n_features)`, during the `fit` step.
```

Reply:
I seem to have misunderstood this. It seems that there is no `fit` and only a `transform`, that can be used both to center the "test" K (K(X, Y) and the "train" K (K(X, X)). Something to clarify a little bit in the doc then ?